tests/server/resource_deploy_performance.py

```diff
@@ -103,214 +108,265 @@ async def run_batch(index):
 
     print(json.dumps(global_registry().dump_metrics(), indent=4))
 
+    key_query = """
+    SELECT DISTINCT ra.* FROM public.resource as r INNER JOIN public.resourceaction_resource as jt
+                        ON r.environment = jt.environment
+                        AND r.resource_id = jt.resource_id
+                        AND r.model = jt.resource_version
+                    INNER JOIN public.resourceaction as ra
+                        ON ra.action_id = jt.resource_action_id
+                        WHERE
+                            r.environment=$1 AND
+                            ra.environment=$1 AND
+                            resource_type=$2 AND
+                            agent=$3 AND
+                            r.resource_id_value = $4::varchar AND
+                            ra.action=$5
+                        ORDER BY started DESC, action_id DESC LIMIT $6"""
+
+    resource_type = "test::Resource"
+    agent = "agent1"
+    resource_id_value = "key1"
+    ra_action = "deploy"
+    limit = 1000
+
+    async with ResourceAction.get_connection() as con:
+        stmt = await con.prepare(key_query)
+        print(json.dumps(await stmt.explain(env_id, resource_type, agent, resource_id_value, ra_action, limit, analyze=True)))
+
+    query_2 = """
+        SELECT r1.resource_version_id, r1.last_non_deploying_status
+        FROM resource AS r1
+        WHERE r1.environment=$1
+              AND r1.model=$2
+              AND (
+                  SELECT (r2.attributes->'requires')::jsonb
+                  FROM resource AS r2
+                  WHERE r2.environment=$1 AND r2.model=$2 AND r2.resource_version_id=$3
+              ) ? r1.resource_version_id
+    """
+    async with ResourceAction.get_connection() as con:
+        stmt = await con.prepare(query_2)
+        print(json.dumps(await stmt.explain(env_id, 2, "test::Resource[agent1,key=key3],v=1", analyze=True)))
+
 
 """
-Baseline output
+Baseline output iso5-stable
```

Comment:
I think we should add a clarifying comment on what the purpose of this baseline is.

Reply:
I removed it